src/components/Menu/index.js

```diff
@@ -23,37 +23,25 @@ function Menu() {
           <Link to="/visits">All Visits</Link>
         </li>
         <li>
-          <Link to="/visits/details/test-id">View a single Visit</Link>
+          <Link to="/visits/details-id">View a single Visit</Link>
```

Comment:
The config path is "/visits/details/:id".
So the correct link here is "/visits/details/details-id"